core/domain/html_validation_service.py

```diff
@@ -135,6 +135,312 @@ def wrap_with_siblings(tag, p):
 BLOCK_COMPONENT_TAG_NAMES = (
     rte_component_registry.Registry.get_block_component_tag_names())
 
+# See https://perso.crans.org/besson/_static/python/lib/python2.7/encodings/cp1252.py # pylint: disable=line-too-long
```

Comment:
Do we need bypass flag here? (IIRC, a link can take more than 80 chars in line)

Reply:
Yeah the lint checks failed because of it so I added it.